stdlib/2/json.pyi

```diff
@@ -11,7 +11,7 @@ def dumps(obj: Any,
           ensure_ascii: bool = ...,
           check_circular: bool = ...,
           allow_nan: bool = ...,
-          cls: Any = ...,
+          cls: Optional[Type['JSONEncoder']] = ...,
```

Comment:
Quotes are not necessary for forward references. (Same below.)

Reply:
flake8 fails if I remove them:
```
$ flake8 stdlib/2/json.pyi
stdlib/2/json.pyi:14:30: F821 undefined name 'JSONEncoder'
stdlib/2/json.pyi:28:29: F821 undefined name 'JSONEncoder'
stdlib/2/json.pyi:38:30: F821 undefined name 'JSONDecoder'
stdlib/2/json.pyi:51:29: F821 undefined name 'JSONDecoder'
```
I see typeshed's flake8 only targets flake8's default filenames though, so `*.py` only, no `*.pyi`. I wonder if that's on purpose? There aren't that many `*.py` in typeshed, and comments in `.flake8` suggest that stubs should be actually checked.

Reply:
Nevermind, I'll just submit another PR for the flake8 stuff.

Reply:
Well doh, I had a wrong virtualenv active, that's why the confusion.